util/GO9PTERM

```diff
@@ -16,8 +16,14 @@ if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
 	fi
 fi
 
+# The -usb flag is necessary for the system to recognize usb input
```

Comment:
instead of input, say 'devices'. It's not just input.
For people who don't know much about lsusb, why not show a sample command and where 'X' comes from? You say 'X' two times below in your example; is it the same X or a different X?

So show an an example of an lsub command,then show what hostbus=X and hostaddr=Y would be in the command below, i.e., show a sample command.